.github/workflows/notify-triager.yml

```diff
@@ -0,0 +1,69 @@
+name: Notify Triagers
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Notify-triagers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4.1.1
+
+      - name: Check PR Changes for .md files
+        id: md-pr-changes
+        uses: tj-actions/changed-files@v42
+        with:
+          files: |
+            **.md
+
+      - name: Check PR Changes for non-.md files
+        id: non-md-pr-changes
+        uses: tj-actions/changed-files@v42
```

Comment:
Better to use the commit tag here instead of version tag.

Reply:
I am not sure I understand

Reply:
Something like https://github.com/asyncapi/.github/blob/09d72fb0774103e4fd36b81cfac1d15f8a3fbba7/.github/workflows/global-replicator.yml#L27